docs/integrations/sources/kafka.md

```diff
@@ -40,10 +40,15 @@ The Kafka source connector supports the following[sync modes](https://docs.airby
 | Incremental - Append Sync | Yes |  |
 | Namespaces | No |  |
 
+## Supported Format
+   Json - Json value messages 
+   Avro - deserialize Using confluent API. Please refer (https://docs.confluent.io/platform/current/schema-registry/serdes-develop/serdes-avro.html)
+
 ## Changelog
 
 | Version | Date       | Pull Request                                           | Subject                                   |
 | :------ | :--------  | :------------------------------------------------------| :---------------------------------------- |
+| 0.2.0 | 2022-08-22 | [13864](https://github.com/airbytehq/airbyte/pull/13864) | Added Avro format support |
```

Comment:
@sivankumar86 rename `Avro` to `AVRO`, also include in the notes `Support for maximum records to process`